### PR TITLE
[SYCLomatic #524] Add test for cub::DeviceReduce::Min/Max

### PR DESCRIPTION
**Pull request by yihwang-nv**
Signed-off-by: Wang, Yihan <yihan.wang@intel.com>

### CODE REVIEW

**Review by tangjj11 — APPROVED** [review]

File: features/feature_case/cub/cub_device_reduce_max.cu [hunk 1]
```diff
@@ -0,0 +1,73 @@
+// ====------ cub_device_reduce_max.cu-------------------- *- CUDA -* ------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//
+// ===----------------------------------------------------------------------===//
+
+// ====------ cub_device_reduce_min.cu-------------------- *- CUDA -* ------===//
```
Comment (ShengchenJ) [inline features/feature_case/cub/cub_device_reduce_max.cu]: The comment looks duplicated.

**Review by ShengchenJ — CHANGES_REQUESTED** [review]

File: features/feature_case/cub/cub_device_reduce_max.cu (hunk 1 above)
Comment (yihwang-nv) [reply features/feature_case/cub/cub_device_reduce_max.cu]: removed.

**Review by ShengchenJ — COMMENTED** [review]
How about merge the min and max implement into one end-to-end case?

**Comment by yihwang-nv** [comment]
> How about merge the min and max implement into one end-to-end case?

I will add argmin/argmax e2e test later, separate tests make it easier to find test failures, what do you think about?

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**